config-connector/solutions/sql/kpt/mysql-public/Kptfile

```diff
@@ -0,0 +1,29 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: mysql-public
+packageMetadata:
+  shortDescription: configures a public MySQL cluster
+openAPI:
+  definitions:
+    io.k8s.cli.setters.instance-name:
+      description: name of SQL instance
+      x-k8s-cli:
+        setter:
+          name: instance-name
+          value: mysql-public-solution
+          setBy: package-default
+    io.k8s.cli.setters.authorized-network:
+      description: name of authorized network
+      x-k8s-cli:
+        setter:
+          name: authorized-network
+          value: mysql-public-solution-sample
+          setBy: package-default
+    io.k8s.cli.setters.authorized-network-cidr:
+      description: authorized network CIDR range
+      x-k8s-cli:
+        setter:
+          name: authorized-network-cidr
+          value: 130.211.0.0/28
+          setBy: package-default
```

Comment:
Thank you for updating the setters! One setter I'd still suggest to have is the password. TF has the feature to auto-generate the password but KCC doesn't. So customers will still need to set it by themselves to avoid having any security issues.